1pass

```diff
@@ -500,7 +504,11 @@ output_result()
         local pbcopy
         pbcopy=pbcopy
         if [ "$os" == "Linux" ] || [ "$os" == "FreeBSD" ]; then
```

Comment:
repeating this double nested `if` block is a bit ugly.. i can also imagine refactoring to extract the clipboard copy command (full command) into a variable before `check_sanity`.. and then in `check_sanity` just check if the `$0` of that command (the base command) is present..